### DISCUSSION TITLE
✅ 🚀 Feature Request: Provide a way to prevent Tabs from turning to Accordions on mobile - [ED-9306]

### DISCUSSION THREAD

**Discussion by jlemonz** [elementor/elementor#21202] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

Well, the title says it clearly: I need the tabs to stay as tabs even on mobile. I don't need them to become accordeon. I have only 3 small tabs and in each one is different form. Accordeon is too triky for that case.

### Describe the solution you'd like

For the time being, I would be fine if you provide me with the script to turn this behavior off.

### Describe alternatives you've considered

I've started a support ticket, but the support answered me, that it isn't possible and I should raise a request here.

But it seems to me quite strange - I suppose this behavior is managed by a script, which is triggered at some point: all I need is to turn this script off. It shouldn't be that difficult, right? I don't want to add anything, I just want to turn something off.

**Comment by jlemonz** [comment]
see https://github.com/elementor/elementor/issues/7742

**Comment by nicholaszein** [comment]
Hello there!

We have great news! 🙌 

### 📢 We're happy to announce that this feature was released with Elementor v3.14.0! 🥳 

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂

----
PS: Please, in the future, **<ins>do not duplicate already opened issues here</ins>**. If there is already an issue, please feel free to comment there. We will migrate all Feature Requests that are still issues in time, but we can't do that if there are duplicates as it takes even more time to migrate everything. Thanks for understanding. 🙏
